tests/apollo/util/skvbc.py

```diff
@@ -424,22 +424,22 @@ async def send_write_kv_set(self, client=None, kv=None, max_set_size=None, long_
                 max_read_set_size = 0 if self.tracker.no_conflicts else max_set_size
                 read_version = self.tracker.read_block_id()
                 readset = self.readset(0, max_read_set_size)
-            reply = await self.send_kv_set(client, readset, writeset, read_version, long_exec, assert_reply, raise_slowErrorIfAny)
+            reply = await self.send_kv_set(client, readset, writeset, read_version, long_exec, assert_reply, raise_slowErrorIfAny, m_of_n_quorum)
             action.log(message_type="[send_write_kv_set] OK")
             if kv_input is True:
                 return reply
             else:
                 return writeset[0][0],writeset[0][1]
 
-    async def send_kv_set(self, client, readset, writeset, read_version, long_exec=False, reply_assert=True, raise_slowErrorIfAny=True):
+    async def send_kv_set(self, client, readset, writeset, read_version, long_exec=False, reply_assert=True, raise_slowErrorIfAny=True, m_of_n_quorum=None):
```

Comment:
I'd simply call this parameter `quorum`, to highlight it accepts various types of quorums.

Reply:
I matched the already existing name for it. Do you want the change everywhere?

Reply:
No, it's up to you to decide, as I said it's a minor thing.